nni/tools/nnictl/launcher.py

```diff
@@ -81,6 +81,10 @@ def start_rest_server(port, platform, mode, experiment_id, foreground=False, log
         cmds += ['--log_level', log_level]
     if foreground:
         cmds += ['--foreground', 'true']
+    if url_prefix:
+        path_validation(url_prefix)
```

Comment:
maybe validate in `create_experiment` is better?
and if we `resume` or `view` an experiment, should we support reconfigure `url_prefix`?

Reply:
1. Changed.
2. I'm ok for that, since there is no way to change "url_prefix". Any concerns @liuzhe-lz ?